en/ha.rst

```diff
@@ -765,12 +765,12 @@ See the above description of **ha_delay_limit**.
 
 This is the maximum value of the time in which a node's database server process (cub_server) waits a response from another node's replication-log-copy process (copylogdb). The default is 5(seconds). If this value is -1, this means to be infinite wait. It only works with **SYNC** log copy mode(**ha_copy_sync_mode**) parameter. 
 
-**ha_monitor_disk_failure_interval** 
+**ha_check_disk_failure_interval** 
 
-CUBRID judges the disk failure for each time which is set to the value of this parameter. The default is 30, and the unit is second.
+CUBRID judges the disk failure for each time which is set to the value of this parameter. The default is 15, and the unit is second.
 
-*   If the value of **ha_copy_log_timeout** parameter is -1, the value of **ha_monitor_disk_failure_interval** parameter is ignored and the disk failure is not judged.
-*   If the value of **ha_monitor_disk_failure_interval** parameter is smaller than the value of **ha_copy_log_timeout** parameter, the disk failure is judged for each **ha_copy_log_timeout** + 20 seconds.
+*   If the value of **ha_copy_log_timeout** parameter is -1, the value of **ha_check_disk_failure_interval** parameter is ignored and the disk failure is not judged.
```

Comment:
When I briefly checked the codes, the ha_check_disk_failure_interval is used by the cub_master process to decide the time interval of the request for checking disk failures to the cub_server. And the ha_copy_log_timeout is used by the cub_server process at the code to handle the request from copylogdb. So, I don't think there is any correlation between the two system parameters. But, I think you have already checked it. Can you share your opinion?

Reply:
When I check code, I verified that ha_check_disk_failure_interval is tuned at only CS mode and the parameter didn't has relation with ha_copy_log_timeout.
Since I don't know why the parameter is tuned using ha_copy_log_timeout at this mode,
I let remain 2 line (772, 773).
I would like to remove 2 line as your opinion. Do I remove it ?

Reply:
I checked the related codes and you don't need to remove them in the manual. The ha_check_disk_failure_interval system parameter is tuned by being affected the ha_copy_log_timeout system parameter in the prm_tune_parameters(). And I found the related history.
- http://jira.cubrid.org/browse/CUBRIDSUS-11145
- http://jira.cubrid.org/browse/CUBRIDSUS-13796
- The related commit and the way to read the fixed code
  - git log f4b9c8fc0e -1
  - git show f4b9c8fc0e -1

Reply:
Thank you for sharing related issue.
I understood why need to tuning ha_check_disk_failure_interval, if ha_copy_log_timeout is -1.
The 2 line of manual have to keep as your comment.